src/inspector/inspectorhandler.cpp

```diff
@@ -616,44 +573,44 @@ static QList<InspectorCommand> s_commands{
           return obj;
         }},
 #ifdef MVPN_ANDROID
-    InspectorCommand{"android_daemon",
-                     "Send a request to the Daemon {type} {args}", 2,
-                     [](InspectorHandler*, const QList<QByteArray>& arguments) {
-                       auto activity = AndroidVPNActivity::instance();
-                       Q_ASSERT(activity);
-                       auto type = QString(arguments[1]);
-                       auto json = QString(arguments[2]);
-
-                       ServiceAction a = (ServiceAction)type.toInt();
-                       AndroidVPNActivity::sendToService(a, json);
-                       return QJsonObject();
-                     }},
+    InspectorHandler::InspectorCommand{
```

Comment:
Now that we have the ability to register inspector commands from within modules, it may be cleaner to move this into `AndroidUtils`